library/modal/modal.css

```diff
@@ -0,0 +1,47 @@
+.modal {
+  display: none;
+  background-color: #00000034;
```

Comment:
use variable for color

Reply:
Okay sure, thanks for pointing this out!